.env.example

```
@@ -68,5 +67,6 @@ MAIL_TRANSPORT_SMTP_AUTH_PWD=password
 # MAIL_TRANSPORT_DKIM_PRIVATE_PATH=/path/to/privkey.pem
 
 
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)